components/src/core/EmptyState/EmptyState.tsx

```diff
@@ -83,7 +83,7 @@ const EmptyStateRender: React.ForwardRefRenderFunction<unknown, EmptyStateProps>
         <Root
             ref={ref}
             className={cx(defaultClasses.root, classes.root, userClassName)}
-            data-test={'frame'}
+            data-testid={'frame'}
```

Comment:
Why "frame" vs something like "blui-empty-state-root"?  Also I'm a little confused about these test ids; our internal tests won't do this, but app teams can have multiple elements rendered on screen at the same time which would then share an id.   I'm wondering if this will cause any complications down the road.  Will it be our recommendation to adopters that they can also query elements using our internal test ids?  

Can these testing ids be queried as if they are classes?  I know cypress uses [data-cy] which doesn't assume the id is unique.

Reply:
left this as is and didn't change "frame" to something more meaningful. Having duplicate data-testid needs research but having them would require the test to focus on the first data-testid using "firstChild" or second....open to suggestions.